WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1803,3 +1805,12 @@ FileTypeCal         = @FileIconCommonStyle@ background-color:#94385d;
 FileTypeVCard       = @FileIconCommonStyle@ background-color:#666666;
 FileTypeMedia       = @FileIconCommonStyle@ background-color:#9e61ca;
 FileTypeMail        = @FileIconCommonStyle@ background-color:@PrimaryBrandingColor@
+
+#######################################################################
+#   Create Contact-Group Dialog
+#######################################################################
+
+CreateContactGroupDialog        =
```

Comment:
Got it. Changed grouping of rules declarations. 
https://github.com/Zimbra/zm-web-client/pull/28/commits/bd5cd40332925558ca17922e9e8681d7061ae28b